book/docs/setup/editors/intelliJ-Windows.md

```diff
@@ -0,0 +1,79 @@
+# IntelliJ IDEA setup on Windows for Plutus
+
+### Credits
+- https://github.com/ganeshnithyanandam
+- @ganesh#9022
+
+### My review
+I have been trying to figure out an optimal IDE setup for working with Plutus smart contracts. I have come to like the following setup on Windows, using the IntelliJ IDE, its HaskForce plugin and Cygwin. All in all, a fairly good IDE experience. 
+
+There are some trade-offs, which you can assess from the pros and cons given below.
+
+**Pros**: 
+   - Does not need WSL on Windows (WSL is still needed if you want to launch plutus playground due to its dependency on nix-shell) 
+   - Haskell syntax highlighting
+   - View and navigate to function definitions
+   - One IDE experience if you work on IntelliJ for other projects
+
+**Cons**:
+   - Code completion is limited to the same file. _(Maybe there is a workaround for this.)_
+   - Certain sections of code causes navigation not work for all code after it in the file. eg: Saw this happen in Value.hs 
+ 
+
+### Setup steps
+1. Install ghc 8.10.4 from [here](https://www.haskell.org/ghc/download_ghc_8_10_4.html#windows64)
+   
+  _Skip your OS drive if you want to space there_
```

Comment:
Hi! Thanks for the PR! I don't quite understand what this line means. Would you be able to reword it, please? Thanks!